### ISSUE TITLE
Not resolving aliases from "browser" field in package.json

### ISSUE THREAD

**Issue by ideriabin** [arian/partition-bundle#8]
I'm trying to get **partition-bundle** to work but it doesn't seem to resolve module names to their actual paths defined in the `browser` fiels in `package.json`. Here is my bundle map:

``` json
{
  "jquery-common.js": [
    "jquery",
    "jquery.cookie",
    "jquery.fancybox",
    "jquery.galleria",
    "jquery.raty",
    "jquery.tabscroll"
  ]
}
```

In `package.json` there are actual library paths defined in the `browser` field:

``` json
{
  "browser": {
    "jquery": "../../../core/third_party/jquery/jquery-1.10.2.min.js",
    "jquery.cookie": "../../../core/third_party/jquery/jquery.cookie.min.js",
    "jquery.fancybox": "../../../core/third_party/jquery/fancybox/jquery.fancybox.pack.js",
    "jquery.galleria": "../../../core/third_party/jquery/galleria-1.2.9.min.js",
    "jquery.raty": "../../../core/third_party/jquery/jquery.raty.js",
    "jquery.tabscroll": "../../../core/third_party/jquery/tabscroll.js"
  }
}
```

I'm trying to generate bundle files by running:

``` bash
browserify -p [ partition-bundle --map bundle.json --output _browserify/bundle ] 
```

…and get an error message:

```
Error: Cannot find module '<path>/jquery.cookie' from '<path>'
    at <path>/node_modules/browserify/node_modules/resolve/lib/async.js:55:21
    at load (<path>/node_modules/browserify/node_modules/resolve/lib/async.js:69:43)
    at onex (<path>/node_modules/browserify/node_modules/resolve/lib/async.js:92:31)
    at <path>/node_modules/browserify/node_modules/resolve/lib/async.js:22:47
    at Object.oncomplete (fs.js:108:15)
```

…which, as I understand, means that it didn't figure out the actual library path and haven't found the file within `cwd`. Is there a way to make it aware of `browser` field definitions?

**Comment by arian** [comment]
This should be fixed no in version 1.2.0

**Comment by ideriabin** [comment]
Thanks! :+1:
